word-search.py

```diff
@@ -229,8 +229,39 @@ def checkWord():
     # checks if user found a correct word
     return
 
-def printWordList():
-    # prints all possible words to be found at game end
+def printWordList(gridWordList, foundWords):
+    # Prints all possible words to be found at game end
+    # Reveals player's answers and possible answers
+    
+    from collections import defaultdict
```

Comment:
Import statements should always be declared at the top of the file. Please move to line 8